Casks/magiccap.rb

```diff
@@ -0,0 +1,12 @@
+cask 'magiccap' do
+  version '2.0.0'
+  sha256 :no_check
```

Comment:
This needs to be set.

Reply:
```suggestion
  sha256 '8cebb0061062e03e54a71ff120a6c658a41bbf8952aadfd2945fd76808dc2cb6'
```

brew cask audit output:
`
➜ brew cask audit --download magiccap
==> Downloading https://github.com/magiccap/MagicCap/releases/download/v2.0.0/magiccap-mac.dmg
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/142702658/f8bc2480-
######################################################################## 100.0%
==> Verifying SHA-256 checksum for Cask 'magiccap'.
audit for magiccap: passed
`